src/SB/Game/zNPCTypeDutchman.cpp

```diff
@@ -207,6 +207,20 @@ S32 zNPCGoalDutchmanCaught::Exit(float dt, void* updCtxt)
     return xGoal::Exit(dt, updCtxt);
 }
 
+
+
+
+S32 zNPCGoalDutchmanDeath::Enter(float dt, void* updCtxt)
```

Comment:
The `float` should be changed to `F32` to match our types (which were how Heavy Iron defined their types in the original codebase)